packages/swap/src/routes/__tests__/__snapshots__/swap.test.ts.snap

```diff
@@ -136,7 +136,7 @@ exports[`server > GET /swaps/:id > retrieves boost details when swap was boosted
   "depositChannelBrokerCommissionBps": 0,
   "depositChannelCreatedAt": 516000,
   "depositChannelExpiryBlock": "265",
-  "depositChannelMaxBoostFeeBps": 30,
+  "depositChannelMaxBoostFeeBps": 7,
```

Comment:
changed because value in `DepositBoosted` overwrites value from `SwapDepositAddressReady`